content/pt-br/docs/concepts/windows/intro.md

```diff
@@ -0,0 +1,286 @@
+---
+title :Contêineres do Windows no Kubernetes
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+Os aplicativos do Windows constituem uma grande parte dos serviços e aplicativos executados em muitas organizações. [Windows containers](https://aka.ms/windowscontainers)
+fornecem uma maneira de encapsular processos e dependências de pacotes, facilitando o uso de práticas de DevOps e seguindo padrões nativos de nuvem para aplicativos do Windows.
+
+As organizações com investimentos em aplicativos baseados em Windows e aplicativos baseados em Linux não precisam procurar orquestradores separados para gerenciar suas cargas de trabalho, levando a um aumento eficiências operacionais em suas implantações, independentemente do sistema operacional.
+
+<!-- body -->
+
+## Windows nodes em Kubernetes
+
+Para habilitar a orquestração de contêineres do Windows no Kubernetes, inclua nós do Windows em seu cluster Linux existente. Agendamento de contêineres do Windows em {{< glossary_tooltip text="Pods" term_id="pod" >}} em O Kubernetes é semelhante ao agendamento de contêineres baseados em Linux.
+
+Para executar contêineres do Windows, seu cluster Kubernetes deve incluir vários sistemas operacionais. Embora você só possa executar o {{< glossary_tooltip text="control plane" term_id="control-plane" >}} no Linux,
+você pode implementar nós do trabalhador executando Windows ou Linux.
+
+Janelas {{< glossary_tooltip text="nodes" term_id="node" >}} são
+[supported](#windows-os-version-support) desde que o sistema operacional seja o Windows Server 2019.
```

Comment:
This sentence is looking strange. I think Windows here refers to the Windows OS and thus should not be translated. Node should be translated as "nó". Supported should be translated as "suportado".